tests/test_routes.py

```diff
@@ -199,6 +199,74 @@ def test_delete(self):
         self.assertEqual(len(response.data), 0)
 
     def test_delete_not_found(self):
-        """It should Delete a promotion and return 204 if not found."""
+        """It should Delete a promotion and return 404 if not found."""
```

Comment:
Same as the above

Reply:
@tanaytrivedi this still needs to be changed to 204, not 404